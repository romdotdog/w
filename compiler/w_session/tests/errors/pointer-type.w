fn foo(): *i32 {}
fn bar(): **************************************************i32 {}