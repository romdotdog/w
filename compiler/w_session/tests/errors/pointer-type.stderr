tests/errors/pointer-type.w:1:12: error: type mismatch
fn main(): *i32 {}
           ^^^^
